### ISSUE TITLE
`throwOnError` does not work with Angular

### ISSUE THREAD

**Issue by robingenz** [TanStack/query#7022]
### Describe the bug

The `throwOnError` option is ignored in the following call:

```ts
injectQuery(() => ({
  queryKey: ['tasks', '5'],
  queryFn: () => {
    throw new Error('Task not found');
  },
  retry: false,
  throwOnError: true,
}));
```

No error is thrown. Instead, the `error` property is set.

### Your minimal, reproducible example

Will be added later

### Steps to reproduce

1. Run the above code

### Expected behavior

An error should be thrown.

### How often does this bug happen?

None

### Platform

- MacOS
- Chrome 122.0.6261.94

### Tanstack Query adapter

angular-query

### TanStack Query version

5.24.6

**Comment by TkDodo** [comment]
@arnoud-dv FYI

**Comment by arnoud-dv** [comment]
Thanks for reporting this. Looking at other adapters some simple code needs to be added to the Angular adapter to support this.

I did just now put it on my todo list but I am lacking time. If you would want to submit a PR with a unit test that would be welcome!

**Comment by arnoud-dv** [comment]
Fixed in [5.26.2](https://github.com/TanStack/query/releases/tag/v5.26.2)
